src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx

```diff
@@ -69,12 +69,20 @@ Not every language or logging framework is supported yet. The following are our
 * [Python logs in context procedures](/docs/logs/logs-context/configure-logs-context-python) for agent [v7.12.0.176](/docs/release-notes/agent-release-notes/python-release-notes)
 * [Ruby logs in context procedures](/docs/logs/logs-context/configure-logs-context-ruby) for agent [v8.6.0 or higher](/docs/release-notes/agent-release-notes/ruby-release-notes)
 
-<Callout variant="tip">
-  APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
-</Callout>
-
 If your APM agent doesn't support our automatic logs in context solution yet, you can continue to use our [manual logs in context solutions](/docs/logs/logs-context/logs-in-context), and forward your logs via our infrastructure agent or supported third-party forwarder.
 
+## Known limitations [#limitations]
+
+APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
+
+Here are some additional known limitations:
+
+* Startup logs or host information are not available. This is because logs are not collected until after the app is loaded.
+* If you are using Kubernetes, the logs will not get decoration from the Kubernetes API. This is because the logs never touch the host or exist in a place where the API can be called.
+* When an APM event is triggered, you may not see all stack traces in the associated logs in context for Java or .NET agents. As a workaround, you can change your drop filter rules.
```

Comment:
Is this APM event related? Or should this read "When an exception is thrown in your application..."